[2024-02-21T07:38:07.604+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-02T00:00:00+00:00 [queued]>
[2024-02-21T07:38:07.629+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-02T00:00:00+00:00 [queued]>
[2024-02-21T07:38:07.629+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:38:07.652+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-02-02 00:00:00+00:00
[2024-02-21T07:38:07.663+0000] {standard_task_runner.py:60} INFO - Started process 109 to run task
[2024-02-21T07:38:07.669+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-02-02T00:00:00+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmpqg3dd4x9']
[2024-02-21T07:38:07.673+0000] {standard_task_runner.py:88} INFO - Job 75: Subtask get_mail
[2024-02-21T07:38:07.739+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-02-02T00:00:00+00:00 [running]> on host 893865381b0d
[2024-02-21T07:38:07.876+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-02-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-02T00:00:00+00:00'
[2024-02-21T07:38:07.877+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 196, in execute
    context_merge(context, self.op_kwargs, templates_dict=self.templates_dict)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/context.py", line 261, in context_merge
    context.update(*args, **kwargs)
  File "/usr/local/lib/python3.8/_collections_abc.py", line 837, in update
    for key, value in other:
ValueError: too many values to unpack (expected 2)
[2024-02-21T07:38:07.900+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extract_email, task_id=get_mail, execution_date=20240202T000000, start_date=20240221T073807, end_date=20240221T073807
[2024-02-21T07:38:07.922+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 75 for task get_mail (too many values to unpack (expected 2); 109)
[2024-02-21T07:38:07.974+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-21T07:38:08.020+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-02-21T07:59:38.963+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-02T00:00:00+00:00 [queued]>
[2024-02-21T07:59:38.971+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-02T00:00:00+00:00 [queued]>
[2024-02-21T07:59:38.972+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:59:38.981+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-02-02 00:00:00+00:00
[2024-02-21T07:59:38.987+0000] {standard_task_runner.py:60} INFO - Started process 371 to run task
[2024-02-21T07:59:38.989+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-02-02T00:00:00+00:00', '--job-id', '130', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmp_lb7y6ab']
[2024-02-21T07:59:38.991+0000] {standard_task_runner.py:88} INFO - Job 130: Subtask get_mail
[2024-02-21T07:59:39.025+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-02-02T00:00:00+00:00 [running]> on host 9bab6a0db245
[2024-02-21T07:59:39.082+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-02-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-02T00:00:00+00:00'
[2024-02-21T07:59:41.737+0000] {ImapSmtp.py:839} INFO - IMAP search: 'gmail:'Message from RPA Python' after:15-12-2023'
[2024-02-21T07:59:44.591+0000] {logging_mixin.py:188} INFO - Message from RPA Python0
[2024-02-21T07:59:44.593+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-142.txt')
[2024-02-21T07:59:44.594+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:44.594+0000] {logging_mixin.py:188} INFO - Message from RPA Python1
[2024-02-21T07:59:44.595+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-143.txt')
[2024-02-21T07:59:44.596+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:44.596+0000] {logging_mixin.py:188} INFO - Message from RPA Python2
[2024-02-21T07:59:44.597+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-144.txt')
[2024-02-21T07:59:44.597+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:44.597+0000] {logging_mixin.py:188} INFO - Message from RPA Python3
[2024-02-21T07:59:44.598+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-145.txt')
[2024-02-21T07:59:44.598+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:44.598+0000] {logging_mixin.py:188} INFO - Message from RPA Python4
[2024-02-21T07:59:44.599+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-146.txt')
[2024-02-21T07:59:44.599+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:44.600+0000] {logging_mixin.py:188} INFO - Message from RPA Python5
[2024-02-21T07:59:44.600+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-147.txt')
[2024-02-21T07:59:44.600+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:44.601+0000] {logging_mixin.py:188} INFO - Message from RPA Python6
[2024-02-21T07:59:44.602+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-148.txt')
[2024-02-21T07:59:44.602+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:44.602+0000] {logging_mixin.py:188} INFO - Message from RPA Python7
[2024-02-21T07:59:44.603+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-149.txt')
[2024-02-21T07:59:44.603+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:44.603+0000] {logging_mixin.py:188} INFO - Message from RPA Python8
[2024-02-21T07:59:44.604+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-150.txt')
[2024-02-21T07:59:44.604+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:44.604+0000] {logging_mixin.py:188} INFO - Message from RPA Python9
[2024-02-21T07:59:44.605+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-151.txt')
[2024-02-21T07:59:44.605+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:45.218+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-21T07:59:45.231+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=extract_email, task_id=get_mail, execution_date=20240202T000000, start_date=20240221T075938, end_date=20240221T075945
[2024-02-21T07:59:45.297+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-21T07:59:45.312+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
